WEEK1 SOLUTION/script3.py

```diff
@@ -0,0 +1,12 @@
+#write a script to check number is palindrome or not..!

```

Comment:
Here, in this program according to definition you first need to check whether the inputed number is integer number Or not..? So, where it is...? Can u write any one way to do that..?